python/ray/autoscaler/_private/kuberay/run_autoscaler.py

```diff
@@ -44,12 +44,12 @@ def run_kuberay_autoscaler(cluster_name: str, cluster_namespace: str):
                     "--skip-version-check",
                 ]
             )
-            # Logging is not ready yet. Print to stdout for now.
-            print("The Ray head is ready. Starting the autoscaler.")
+            logger.info("The Ray head is ready. Starting the autoscaler.")
             break
         except subprocess.CalledProcessError:
-            print("The Ray head is not yet ready.")
-            print(f"Will check again in {BACKOFF_S} seconds.")
+            logger.info(
```

Comment:
nit: Should we use higher-level logging, considering it's an exception?

Reply:
updated

Reply:
some context: It is possible that the health check timeout will occur in regular cases in the future, because I plan to separate the autoscaler into its own Pod instead. Currently, `warning` makes sense, and we can revisit after we separate the autoscaler.